src/cli.rs

```diff
@@ -20,32 +20,43 @@ use crate::normal_path::NormalPath;
 ///
 /// Load `cabal v2-repl` and watch for changes in `src`:
 ///
-///     ghciwatch
+/// ```bash
```

Comment:
These examples don't use triple backticks because they're rendered in `ghciwatch --help` as well as being used for the online docs... I want to get a better system in place but don't really know what it should look like

Reply:
Without this, the doctests seem to fail locally for me. Is that expected?

Reply:
I'm not sure there's any actual doctests? We may need some attribute to tell it to not run code examples here.